frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/cdrom-row.tsx

```diff
@@ -50,8 +51,8 @@ export const CDRomRow: React.FC<CDRomRowProps> = ({
     ejected,
     isInVM,
   } = cd;
-  const { upstream } = WINTOOLS_CONTAINER_NAMES;
-
+  const { upstream, downstream } = WINTOOLS_CONTAINER_NAMES;
```

Comment:
I would avoid the `downstream` terminology and make this logic more indirect (generic).

If we can not retrieve the names in a configurable way (indeed, it would be complicated), the `WINTOOLS_CONTAINER_NAMES` can be a map of `[BRAND_NAME]: [name]` with a `default` key conforming the recent `upstream`.

And please move these constants and logic around them to an isolated file and utility functions.

Reply:
fixed @mareklibra